sources/core/Xenko.Core.Mathematics/ColorBGRA.cs

```diff
@@ -499,7 +499,7 @@ public static void Modulate(ref ColorBGRA left, ref ColorBGRA right, out ColorBG
         /// <returns>The modulated color.</returns>
         public static ColorBGRA Modulate(ColorBGRA left, ColorBGRA right)
         {
-            return new ColorBGRA((left.R * right.R) >> 8, (left.G * right.G) >> 8, (left.B * right.B) >> 8, (left.A * right.A) >> 8);
+            return new ColorBGRA((byte)((left.R * right.R) >> 8), (byte)((left.G * right.G) >> 8), (byte)((left.B * right.B) >> 8), (byte)((left.A * right.A) >> 8));
```

Comment:
Same

Reply:
That's what `>> 8` is supposed to do here. But it's wrong because it divides by 256 instead of 255.

Reply:
Right, sorry I thought this was same mistake as in `Color.cs`.
Anywway, should be / 255 as you said.

Reply:
Hm, well, I could add it once I add the in keywords soon.
Let's wait for the C# 7.2 PR first. (Which I'm going to upload any moment now)